flink-end-to-end-tests/flink-tpcds-test/src/main/java/org/apache/flink/table/tpcds/TpcdsTestProgram.java

```diff
@@ -157,34 +166,33 @@ private static TableEnvironment prepareTableEnv(String sourceTablePath, Boolean
                 table -> {
                     TpcdsSchema schema = TpcdsSchemaProvider.getTableSchema(table);
-                    CsvTableSource.Builder builder = CsvTableSource.builder();
-                    builder.path(sourceTablePath + FILE_SEPARATOR + table + DATA_SUFFIX);
-                    for (int i = 0; i < schema.getFieldNames().size(); i++) {
-                        builder.field(
-                                schema.getFieldNames().get(i),
-                                TypeConversions.fromDataTypeToLegacyInfo(
-                                        schema.getFieldTypes().get(i)));
-                    }
-                    builder.fieldDelimiter(COL_DELIMITER);
-                    builder.emptyColumnAsNull();
-                    builder.lineDelimiter("\n");
-                    CsvTableSource tableSource = builder.build();
-                    ConnectorCatalogTable catalogTable =
-                            ConnectorCatalogTable.source(tableSource, true);
-                    tEnv.getCatalog(tEnv.getCurrentCatalog())
-                            .ifPresent(
-                                    catalog -> {
-                                        try {
-                                            catalog.createTable(
-                                                    new ObjectPath(
-                                                            tEnv.getCurrentDatabase(), table),
-                                                    catalogTable,
-                                                    false);
-                                        } catch (Exception e) {
-                                            throw new RuntimeException(e);
-                                        }
-                                    });
+                    String filePath = sourceTablePath + FILE_SEPARATOR + table + DATA_SUFFIX;
+
+                    tEnv.createTable(
+                            table,
+                            TableDescriptor.forConnector(FILE_CONNECTOR_NAME)
+                                    .schema(
+                                            Schema.newBuilder()
+                                                    .fromFields(
+                                                            schema.getFieldNames(),
+                                                            schema.getFieldTypes())
+                                                    .build())
+                                    .format(
+                                            FormatDescriptor.forFormat(CSV_FORMAT)
+                                                    .option(
+                                                            CsvFormatOptions.FIELD_DELIMITER,
+                                                            COL_DELIMITER)
+                                                    .option(
+                                                            CsvFormatOptions.IGNORE_PARSE_ERRORS,
```

Comment:
Looks to me this config it is not needed to be `true` because `ignoreParseErrors()` was not invoked on the old `CsvTableSource`.
Correct me if parse errors do happen in this case.

Reply:
you are right, **_CsvFormatOptions.IGNORE_PARSE_ERRORS_** is not needed. I run tpcds without this config, the job still works well.